test/jdk/javax/swing/text/html/CSS/MissingColorNames.java

```diff
@@ -41,15 +39,15 @@
 import static javax.swing.text.html.CSS.Attribute.BORDER_LEFT_COLOR;
 import static javax.swing.text.html.CSS.Attribute.BORDER_RIGHT_COLOR;
 
-public class JDK8292276MissingColorNamesInCSS {
+public class MissingColorNames {
 
     // The CSS 'color' property accepts <name-color>, 'transparent' keyword <rgb()>, <rgba> values.
     // - 'cyan' is the missing <name-color> keyword that originates the PR JDK8292276 :
     //   Missing Color Names In CSS.
     //   'cyan' keyword, as 131 <name-color> keywords also defined in CSS Color Module
     //   Level 4, are not referenced in CSS.java.
```

Comment:
Maybe we should simplify the description?

- 'cyan' is one of the missing <name-color> keywords which are defined in CSS Color Module Level 4.